documentation/modules/cruise-control/proc-generating-optimization-proposals.adoc

```diff
@@ -125,7 +125,30 @@ Status:
 ----
 +
 The properties in the `Optimization Result` section describe the pending cluster rebalance operation. 
-For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals]. 
+For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals].
+
+.Insufficient CPU capacity
+
+If a Kafka cluster is overloaded in terms of CPU utilization, you might see an insufficient CPU capacity error in the `KafkaRebalance` status. 
+This might be because of excluded topics configuration (`excludedTopics`). 
+Optimization proposals ignore excluded topics, but CPU utilization calculations consider their replicas.
```

Comment:
I am fine with leaving the mention of excludeTopics up here in the first paragraph to prevent user confusion but I think we should rephrase so something like:

> It's worth noting that this utilization value is unaffected by the excluded topics configuration (excludedTopics). Although optimization proposals will not reassign replicas of excluded topics, their load is still considered in the utilization calculation.

Reply:
Much better, thanks!